brave/brave-core
# Integrated font style, size and content style API on speedreader UI

Resolves https://github.com/brave/brave-browser/issues/26117

This PR integrates font size/style, content style, active/inactive reading mode APIs to the Speedreader WebUI

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Start the browser with `--enable-features=SpeedreaderPanelV2` 
2. Go to an article on nytimes or other verge

Open the Speedreader panel and test if all controls are functional
Also, start two browser windows and change setting in one window to ensure if change propogates on both instances
